packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -360,23 +359,9 @@ class Treemap extends StatelessWidget {
     if (rootNode.children.isNotEmpty) {
       return Padding(
         padding: const EdgeInsets.all(1.0),
-        child: buildSelectable(
-          selectable: !isOutermostLevel,
-          child: Column(
-            children: [
-              if (height > minHeightToDisplayTitleText) buildTitleText(context),
-              Expanded(
-                child: Treemap.fromNodes(
-                  nodes: rootNode.children,
-                  levelsVisible: levelsVisible,
-                  onRootChangedCallback: onRootChangedCallback,
-                  width: width,
-                  height: height,
-                ),
-              ),
-            ],
-          ),
-        ),
+        child: isOutermostLevel
+            ? buildTreemapFromNodes(context)
```

Comment:
store the output of buildTreemapFromNodes in a var instead of calling it twice

Reply:
Done